### PR TITLE
Python3 : v2v cases

### PR DESCRIPTION
**Pull request by chloerh**
Signed-off-by: Haijiao Zhao <haizhao@redhat.com>

### CODE REVIEW

File: v2v/tests/src/convert_from_file.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def run(test, params, env):
     """
     convert specific kvm guest to rhev
     """
-    for v in params.itervalues():
+    for v in list(params.values()):
```
Comment (chunfuwen) [inline v2v/tests/src/convert_from_file.py]: use six module, see below example:
https://github.com/autotest/tp-libvirt/pull/1445/files

Comment (chloerh) [reply v2v/tests/src/convert_from_file.py]: This way works fine for v2v cases, I don't see any reason to introduce an unnecessary module.

Comment (chunfuwen) [reply v2v/tests/src/convert_from_file.py]: This is our unified way to deal with this issue in whole tp-libvirt project

Comment (chloerh) [reply v2v/tests/src/convert_from_file.py]: In v2v test cases, this is simply a parameters existence check. There's no need to make the params iterable. Still, I don't see the necessity even with the unified way.

**Pull request merged**